File: cpp/1672_RichestCustomerWealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int high = 0;
        for (vector<int> account : accounts) {
            int curr = 0;
            for (int n : account) {
                curr += n;
            }
            high = max(curr, high);
        }
        return high;
    }
};

File: swift/1672_RichestCustomerWealth.swift
class Solution {
    func maximumWealth(_ accounts: [[Int]]) -> Int {
        var highestSum = 0
        for account in accounts {
            var currSum = 0
            for n in account {
                currSum += n
            }
            highestSum = max(currSum, highestSum)
        }
        return highestSum
    }
}
